src/select/src/Select.tsx

```diff
@@ -263,27 +264,30 @@ export default defineComponent({
         }
       }
     })
+    function getMergedOption (value: ValueAtom): SelectBaseOption | undefined {
+      const remote = props.remote
+      const { value: memoValOptMap } = memoValOptMapRef
+      const { value: valOptMap } = valOptMapRef
+      const { value: wrappedFallbackOption } = wrappedFallbackOptionRef
+      if (valOptMap.has(value)) {
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+        return valOptMap.get(value)!
+      } else if (remote && memoValOptMap.has(value)) {
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+        return memoValOptMap.get(value)!
+      } else if (wrappedFallbackOption) {
+        return wrappedFallbackOption(value)
+      }
```

Comment:
```suggestion
      }
      return null
```

这么做可以皆大欢喜了？省的在外面 `|| null`